packages/itwinui-react/src/core/DatePicker/DatePicker.tsx

```diff
@@ -665,6 +674,22 @@ export const DatePicker = React.forwardRef((props, forwardedRef) => {
                 {weekDays.map((weekDay, dayIndex) => {
```

Comment:
One more consideration (non-blocking):

In some cases, using this prop results in empty space at the **top** rather than bottom. 

![image](https://github.com/iTwin/iTwinUI/assets/9084735/f2e50ca0-a754-4634-b353-537825dde783)

https://github.com/iTwin/iTwinUI/blob/e739baa6edb34c7b62a66f606b9e5aa781a51362/packages/itwinui-react/src/core/DatePicker/DatePicker.tsx#L342-L345

I'm not sure why it was designed this way originally, but I think it would make more sense to only show empty space at the bottom.